contracts/src/periphery/cert/EigenDACertTypes.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+/// @title EigenDACertTypes
+/// @notice This library defines the types for each EigenDA certificate version.
+/// @dev It is required that RBN be located in positions 64:96 (padded) in the ABI encoded certificate.
+///      The reason for this is for backwards compatibility with EigenDACertVerifierV2.
```

Comment:
is it? how so?

Reply:
I think this is what @ethenotethan mentioned? I'm personally not really sure. It's not backwards compatible in solidity of course. maybe I'll just remove the comment?

Reply:
I think so yeah. Unless @ethenotethan has a really good argument, I tend to think we should mention V2 as little as possible given the versioning it confusing, the naming is bad, we didnt have a struct for it, we plan to nuke it, etc.